eng/pipelines/runtime-official.yml

```diff
@@ -98,6 +98,7 @@ stages:
       - Android_x86
       - Android_arm
       - Android_arm64
+      - MacCatalyst_x64
```

Comment:
add `MacCatalyst_arm64` ?

Reply:
I think that's blocking on arm64 LLVM packages

Reply:
Blocking on a different issue, https://github.com/dotnet/runtime/pull/47891